terraform/constellation-cluster/install-constellation.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+if [[ -f ./constellation ]]; then
+  echo "constellation CLI is already available."
+  exit 0
+fi
+
+echo "Fetching constellation ${VERSION}"
+OS=$(uname -s)
+ARCH=$(uname -m)
+VERSION="latest"
+URL=""
```

Comment:
VERSION isn't set when echoed.

Having VERSION as a var here indicates that one could replace `latest` with a version, but that won't work because the URLs are constructed differently.